packages/office-ui-fabric-react/src/components/Persona/Persona.Props.ts

```diff
@@ -167,5 +167,6 @@ export enum PersonaInitialsColor {
   black = 11,
   orange = 12,
   red = 13,
-  darkRed = 14
+  darkRed = 14,
+  overflowButton = 15
```

Comment:
`overflowButton` color isn't defined here.

https://github.com/OfficeDev/office-ui-fabric-react/blob/7f4ece990b065b1d43ec71cd4d4615b5ede0d082/packages/office-ui-fabric-react/src/components/Persona/PersonaConsts.ts#L30

Reply:
this const is the missing piece I was looking for, thanks! I'll take a look at how this piece fits in the puzzle.